scripts/run-live-build.sh

```diff
@@ -51,6 +67,7 @@ set -o xtrace
 
 export APPLIANCE_VARIANT="$1"
 export APPLIANCE_PLATFORM="$2"
+export RUN_TYPE="$3"
 export ARTIFACT_NAME="$APPLIANCE_VARIANT-$APPLIANCE_PLATFORM"
```

Comment:
Does "ARTIFACT_NAME" need to also contain the "RUN_TYPE" so that we have a unique "build_dir" for each run type? e.g. if we ran the "VmArtifacts" task, then the "UpgradeArtifacts" task, will this cause any problems? The second task will remove the build directory of the first task, right? I don't know if this is a problem, but wanted to ask in case this wasn't considered.

Reply:
This would actually be an issue with the older way (see below comment), but this way only runs each task once (there are no dependencies), so this is fine